### PR TITLE
Report Haskell tests run duration

### PR DESCRIPTION
**Pull request by charlieshanley**

### CODE REVIEW

File: nri-prelude/src/Test/Reporter/Stdout.hs [hunk 1]
```diff
@@ -108,6 +120,8 @@ renderReport results =
                 failures,
             [ red (Text.Colour.underline "TEST RUN FAILED"),
               "\n\n",
+              black <| chunk <| "Duration:  " ++ Text.fromInt elapsed ++ " ms",
```
Comment (stoeffel) [inline nri-prelude/src/Test/Reporter/Stdout.hs]: Should we report in seconds? I bet most tests take longer than a few ms.

File: nri-prelude/src/Test/Reporter/Stdout.hs [hunk 2]
```diff
@@ -172,3 +186,20 @@ testFailure test =
           ++ "This is a bug.\n\n"
           ++ "If you have some time to report the bug it would be much appreciated!\n"
           ++ "You can do so here: https://github.com/NoRedInk/haskell-libraries/issues"
+
+elapsedMilliseconds :: List (Internal.SingleTest Platform.Internal.TracingSpan) -> List (Internal.SingleTest Internal.FailedSpan) -> Int
```
Comment (stoeffel) [inline nri-prelude/src/Test/Reporter/Stdout.hs]: Very nit-picky, but I think this function does too much.
How about:
```hs
elapsedMilliseconds :: List TracingSpan -> Int

spans :: SuiteResult -> List TracingSpan

-- and maybe add something like
-- for rendering the elapsed time into a human readable format not only to ms
formatElapsedTime : Int -> Text
```

**Review by stoeffel — APPROVED** [review]
Nothing blocking a merge just some minor comments.
Thanks for adding this!

File: nri-prelude/src/Test/Reporter/Stdout.hs (hunk 1 above)
Comment (asterite) [reply nri-prelude/src/Test/Reporter/Stdout.hs]: I thought about this but the Elm test output shows it in `ms`. I guess seconds makes sense now that tests are taking about 5 seconds to run, though... I'll change it!

**Comment by asterite** [comment]
@stoeffel I applied your suggestions, thanks! Let me know if it's good to go.

**Review by stoeffel — APPROVED** [review]
LGTM!

thanks

**Comment by asterite** [comment]
@NoRedInkBot r+

**Pull request merged**